docs/frost-page-title.md

```diff
@@ -64,6 +64,30 @@ The `defaultHandler` method takes a url (defaulted to `window.location.hash` or
 ### #updateTitle()
 The `updateTitle` function runs runs the `defaultHandler` and the hanslers built up by routes using the mixin and joins them with either a "|" or whatever you set in your config for `APP['frost-page-title'].delimiter` (see example above).
 
+## Details page title factory util
```

Comment:
In the previous section before this one `hanslers` should be ` handlers`

Reply:
dangit - I did a find and replace for that, but must have left off the 'g'.